src/views/virtualmachinesinstance/details/tabs/details/virtual-machines-instance-details-tab.scss

```diff
@@ -0,0 +1,10 @@
+.VirtualMachinesInstanceDetailsTab {
+  .link-icon {
```

Comment:
no need for custom css.

you can take classnames from console

for example
`<PencilAltIcon className="co-icon-space-l pf-c-button-icon--plain" />`